hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestHoodieDataSource.java

```diff
@@ -1046,6 +1046,7 @@ void testBulkInsert(String indexType, boolean hiveStylePartitioning) {
         .option(FlinkOptions.PATH, tempFile.getAbsolutePath())
         .option(FlinkOptions.OPERATION, "bulk_insert")
         .option(FlinkOptions.WRITE_BULK_INSERT_SHUFFLE_INPUT, true)
+        .option(FlinkOptions.WRITE_BULK_INSERT_SORT_INPUT_BY_RECORD_KEY, true)
         .option(FlinkOptions.INDEX_TYPE, indexType)
```

Comment:
parameterize the test or add a new one

Reply:
I have modified the operate name and added a new test. please help to review it again @danny0405